gateway/src/apicast/policy/url_rewriting/apicast-policy.json

```diff
@@ -107,6 +144,57 @@
               ],
               "default": "plain"
             }
+          },
+          "methods": {
```

Comment:
It'll be nice if a new definition for methods can be done, we're trying to implement that because make things easier to follow and less verbose. As a example: 
https://github.com/3scale/APIcast/blob/master/gateway/src/apicast/policy/jwt_claim_check/apicast-policy.json#L13

Reference
https://json-schema.org/understanding-json-schema/structuring.html

Reply:
fixed